README.md

```diff
@@ -78,9 +82,20 @@ basesystem for more efficient caching.
 An optional argument is passed as options to `osc` and can be used to prefer
 local packages: `rake "osc:build[-p /var/tmp/YaST:Head/openSUSE_Factory]"`
 
+This task checks whether all changes are committed to Git (using the
+[check:committed](#checkcommitted) task). If you want to build the package
+with your local changes without committing to Git set `COMMIT_CHECK` to `0`:
```

Comment:
I am not sure how it will work with removed/added files.

Reply:
Good point... :worried:

Reply:
So please document it in README as caveeat as it can surprise users. Otherwise change looks good.

Reply:
Updated.